schedule/staging/RAID1@s390x-staging.yaml

```diff
@@ -0,0 +1,66 @@
+---
+name:           RAID1_s390x@s390x-staging
+description:    >
+  Test for RAID1
+  Installation of RAID1 using expert partitioner.
+vars:
+  RAIDLEVEL: 1
+  YUI_REST_API: 1
+schedule:
+  - installation/bootloader_start
+  - installation/setup_libyui
+  - installation/welcome
+  - installation/accept_license
+  - installation/scc_registration
+  - installation/addon_products_sle
+  - installation/system_role
+  - installation/partitioning
+  - installation/partitioning/raid_gpt
+  - installation/installer_timezone
+  - installation/hostname_inst
+  - installation/user_settings
+  - installation/user_settings_root
+  - installation/resolve_dependency_issues
+  - installation/installation_overview
+  - installation/disable_grub_timeout
+  - installation/start_install
+  - installation/await_install
+  - installation/logs_from_installation_system
+  - installation/reboot_after_installation
+  - installation/grub_test
```

Comment:
I think it should be `handle_reboot`. Same for the other test suite.

Reply:
Done